vars/infra.groovy

```diff
@@ -272,23 +272,13 @@ void prepareToPublishIncrementals() {
 void maybePublishIncrementals() {
     if (new InfraConfig(env).isRunningOnJenkinsInfra() && currentBuild.currentResult == 'SUCCESS') {
         stage('Deploy') {
-          timeout(300) {
-            node('maven || linux || windows') {
-                timeout(15) {
-                    withCredentials([string(credentialsId: 'incrementals-publisher-token', variable: 'FUNCTION_TOKEN')]) {
-                        if (isUnix()) {
-                            sh '''
-    curl --retry 10 --retry-delay 10 -i -H "Authorization: Bearer $FUNCTION_TOKEN" -H 'Content-Type: application/json' -d '{"build_url":"'$BUILD_URL'"}' "https://incrementals.jenkins.io/" || echo 'Problem calling Incrementals deployment function'
-                            '''
-                        } else {
-                            bat '''
-    curl.exe --retry 10 --retry-delay 10 -i -H "Authorization: Bearer %FUNCTION_TOKEN%" -H "Content-Type: application/json" -d "{""build_url"":""%BUILD_URL%""}" "https://incrementals.jenkins.io/" || echo Problem calling Incrementals deployment function
-                            '''
-                        }
-                    }
-                }
-            }
-          }
+            httpRequest url: 'https://incrementals.jenkins.io/',
+                        httpMode: 'POST'
+                        contentType: 'APPLICATION_JSON',
+                        validResponseCodes: '100:599',
```

Comment:
Will this at least print a warning if it gets an error?

Reply:
should do

Reply:
> 13:57:21  Response Code: HTTP/1.1 403 Forbidden